### PR TITLE
[releases] backfill statistics

### PR DESCRIPTION
**Pull request by dcramer**
#nochanges

### CODE REVIEW

File: src/sentry/south_migrations/0328_backfill_release_stats.py [hunk 1]
```diff
@@ -0,0 +1,895 @@
...
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+        Deploy = orm['sentry.Deploy']
+        Release = orm['sentry.Release']
+        ReleaseCommit = orm['sentry.ReleaseCommit']
+
+        queryset = Release.objects.all()
+        for release in RangeQuerySetWrapperWithProgressBar(queryset):
+            with transaction.atomic():
+                last_deploy = Deploy.objects.filter(
```
Comment (dcramer) [inline src/sentry/south_migrations/0328_backfill_release_stats.py]: these aren't guaranteed correct as theres no lock and they're being generated app-side

tl;dr doesn't matter, its good enough

File: src/sentry/south_migrations/0328_backfill_release_stats.py [hunk 2]
```diff
@@ -0,0 +1,895 @@
...
+                    release=release,
+                ).order_by('-date_finished').first()
+                last_commit = ReleaseCommit.objects.filter(
+                    release=release,
+                ).order_by('order').first()
+                Release.objects.filter(
+                    id=release.id,
+                ).select_for_update().update(
```
Comment (dcramer) [inline src/sentry/south_migrations/0328_backfill_release_stats.py]: would be kind of nice to do this as part of RangeQuerySetWrapper, but could potentially be very expensive

**Review by mattrobenolt — COMMENTED** [review]
As the checklist from Danger says, can we change this to do: https://github.com/getsentry/sentry/blob/master/src/sentry/south_migrations/0312_create_missing_emails.py#L11-L24

Both so we don't hold a long transaction, and two, so if it errors, we see why it errored.

**Review by mattrobenolt — APPROVED** [review]
